lib/config/options/index.ts

```diff
@@ -749,7 +749,7 @@ const options: RenovateOptions[] = [
   {
     name: 'baseBranches',
     description:
-      'An array of one or more custom base branches to be processed. If left empty, the default branch will be chosen.',
+      'List of one or more custom base branches to be processed. Base branches can be defined as exact strings (e.g. `["main"]`) and/or via regex expressions (e.g. `["/^release\\/.*/"]`). If left empty, the default branch will be used.',
```

Comment:
Maybe a bit long for this field?

Reply:
How about this?
```suggestion
      'List of one or more custom base branches defined as exact strings (e.g. `["main"]`) and/or via regex expressions (e.g. `["/^release\\/.*/"]`). If left empty, the default branch will be processed.',
```

Reply:
Maybe put the last part in markdown instead

Reply:
I'd suggest to omit the last part overall. The first two sentences of the [extended description](https://github.com/renovatebot/renovate/blob/main/docs/usage/configuration-options.md#basebranches) already say the same:
> By default, Renovate will detect and process only the repository's default branch. For most projects, this is the expected approach.